packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -90,6 +91,16 @@ module.exports = {
     devtoolModuleFilenameTemplate: info =>
       path.resolve(info.absoluteResourcePath).replace(/\\/g, '/'),
   },
+  optimization: {
+    // Automatically split vendor and commons
+    // https://twitter.com/wSokra/status/969633336732905474
+    splitChunks: {
+      chunks: 'all',
+    },
+    // Keep the runtime chunk seperated to enable long term caching
+    // https://twitter.com/wSokra/status/969679223278505985
+    runtimeChunk: true,
```

Comment:
Does dev config actually use long term caching? I can't see any hashes in script filenames

Reply:
Long term caching is defined on the  http server, by setting far future expire headers. hashes in filenames allows  to flush the cache on the end user.   This is not for long term caching in dev, just having the code split in the same way as in production.